r2/r2/lib/mr_tools/mr_tools.py

```diff
@@ -88,7 +88,7 @@ def __init__(self):
         pass
 
     def process(self, values):
-        raise NotImplemented
```

Comment:
Ah yes, you are completely right. Given that the code currently throws a type error (because `NotImplementedType` is not derived from `BaseException`) I don't think anyone is depending on the current behavior.